platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -182,7 +183,7 @@ void doImport() {
         } else if (version.equals(migratable.getVersion())) {
```

Comment:
❓ Aren't we going to call `doImport()` even in cases where the product version has changed but the migratable hasn't?

Reply:
Yes, you're right 🤔

Reply:
This is not the case anymore since a different import method is called on this class if the product versions are different.